internal/agent/DatabaseTokenManager.go

```diff
@@ -101,20 +103,18 @@ func (rtm *rethinkTokenManager) Verify(token string) error {
 }
 
 // GetUserData returns the user data for the specified token
-func (rtm *rethinkTokenManager) GetUserData(token string) interfaces.UserData {
-	conn := etc.GetConnection()
-	udata, _ := models.GetUserToken(conn, token)
+func (rtm *DatabaseTokenManager) GetUserData(token string) interfaces.UserData {
+	udata, _ := rtm.dbToken.GetUserToken(token)
```

Comment:
Sorry if this is a dumb question, but I don't have the entire context yet: Should we be ignoring this error?

Reply:
The common error cases are `not found` for this specific call. We handle that in the receiver part as a `nil` on userData (which is not found). So basically if that have a error, the `udata` will be nil, then the whole function returns nil as expected.

Reply:
A good practice in this case is to return the error, and the caller checks if the error is some kind of `ErrNotFound`. Even though the common error is not found, it may not be all the cases, and the system should be able to react to the errors correctly. I understand that this change may cause a massive modification do the code base, so that's something I'd save this modification in a backlog for later.